thepeg.spec

```diff
@@ -33,29 +34,44 @@ CXX="$(which %{cms_cxx}) -fPIC"
 CC="$(which gcc) -fPIC"
 PLATF_CONF_OPTS="--enable-shared --disable-static"
 
-case %{cmsplatf} in
-  osx*)
-    LIBQUADMATH="-lquadmath"
-    ;;
-esac
+
+# Update to detect aarch64 and ppc64le
```

Comment:
these should be picked up as sources rather than here so that its more easily understood when the build configuration changes.